config/locales/en.json

```diff
@@ -293,7 +293,13 @@
 	"section1.contact_time_morning": "Morning",
 	"section1.first_name": "First name",
 	"section1.last_names": "Last name(s)",
```

Comment:
If we can, we should avoid `(s)` as it's difficult to scan. Just having `Last name` is cool here.

Reply:
done